x/oracle/types/keys.go

```diff
@@ -25,10 +25,10 @@ const (
 
 var (
 	// OraclesKey defines key to store oracle
-	OraclesKey                      = []byte{0x01}
-	OracleRegistrationsKey          = []byte{0x02}
-	OracleRegistrationVotesKey      = []byte{0x03}
-	OracleRegistrationVotesQueueKey = []byte{0x04}
+	OraclesKey                  = []byte{0x01}
+	OracleRegistrationsKey      = []byte{0x02}
+	OracleRegistrationVotesKey  = []byte{0x03}
```

Comment:
```suggestion
	OracleRegistrationVoteKey  = []byte{0x03}
```
This isn't the part of this PR, but we need to fix this :)

Reply:
I fixed it :)